WordPress/Sniffs/Theme/FileIncludeSniff.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * WordPress Coding Standard.
+ *
+ * @category PHP
+ * @package  PHP_CodeSniffer
+ * @link     https://make.wordpress.org/core/handbook/best-practices/coding-standards/
+ */
+
+/**
+ * WordPress_Sniffs_Theme_FileIncludeSniff.
+ *
+ * WARNING (manual check required) | Check if a theme uses include(_once) or
+ * require(_once) (where they should use get_template_part()). Current implementation
+ * excluded the functions.php file from this check. We may want to continue doing so.
+ *
+ * @category  PHP
+ * @package   PHP_CodeSniffer
+ * @author    khacoder
+ */
+class WordPress_Sniffs_Theme_FileIncludeSniff extends WordPress_AbstractThemeSniff {
+
+	/**
+	 * A list of tokenizers this sniff supports.
+	 *
+	 * @var array
+	 */
+	public $supportedTokenizers = array(
+		'PHP',
+	);
+
+	/**
+	 * Returns an array of tokens this test wants to listen for.
+	 *
+	 * @return array
+	 */
+	public function register() {
+		return array(
+			T_REQUIRE,
```

Comment:
One more: you probably should use `PHP_CodeSniffer_Tokens::$includeTokens` here. You can just straight return it.